solcore/parameter_system/parameter_system.py

```diff
@@ -53,6 +53,7 @@ def __init__(self, sources: Optional[Callable] = None):
             "([A-Z][a-z]*)")
 
     def get_parameter(self, material, parameter, verbose=False, **others):
+
```

Comment:
Why is the a blank line here?

Anyway, since this functionhas changed its behavior - now providing `"In0.2GaAsP0.1"` will not be enough to retrieve the material, the docstring will need to be updated explaining what it expect for inputs.

Reply:
Blank line was accidental... I have updated the docstring and added a DeprecationWarning.